ovos_persona_server/chat.py

```
@@ -0,0 +1,316 @@
+
+    # Convert prompt to a list of messages for persona.chat/stream
+    # The persona expects a list of dicts with 'role' and 'content'
+    messages: List[Dict[str, str]]
+    if isinstance(prompt, str):
+        messages = [{"role": "user", "content": prompt}]
+    elif isinstance(prompt, list) and all(isinstance(p, str) for p in prompt):
+        # If it's a list of strings, join them into a single message.
+        messages = [{"role": "user", "content": "\n".join(prompt)}]
+    elif isinstance(prompt, list) and all(isinstance(p, int) for p in prompt):
+        raise HTTPException(status_code=status.HTTP_500_INTERNAL_SERVER_ERROR,
+                            detail=f"Failed to generate completion: token_ids support not yet implemented")
```

Ruff (0.14.0)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)